tests/fixtures/noqa.py

```diff
@@ -146,6 +146,7 @@ class ManyParents(First, Second, Third, Exception):  # noqa: WPS215
     assert test_function(5)
     assert test_function(6)
 
+some.other[0].field.type.boom  # noqa: WPS219,WPS428
```

Comment:
This should be an assignment, so it won't raise `MeaninglessExpressionViolation`